### PR TITLE
fix(patch): Release v2.0.1

### PR DESCRIPTION
**Pull request by zFernand0**
⚠️ Waiting on the following PRs ⚠️ 
- [x] #1778 
- [x] #1780 
- [x] #1701
- [x] #1792

### CODE REVIEW

File: packages/zowe-explorer/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
...
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
+## `2.0.1`
+
+- BugFix: Fixed adding profiles to the tree view on Theia. [#1774](https://github.com/zowe/vscode-extension-for-zowe/issues/1774)
+- BugFix: Updated Log4js version to resolve initialization problem on Eclipse Che. [#1692](https://github.com/zowe/vscode-extension-for-zowe/issues/1692)
+- BugFix: Fixed dataset upload issue by trimming labels. [#1789](https://github.com/zowe/vscode-extension-for-zowe/issues/1789)
+- BugFix: Fixed duplicate jobs appearing in the jobs view upon making an owner/prefix filter search for extenders. [#1780](https://github.com/zowe/vscode-extension-for-zowe/pull/1780)
+- BugFix: Fixed error displayed when opening a job file for extenders. [#1701](https://github.com/zowe/vscode-extension-for-zowe/pull/1701)
+- BugFix: Improved logging information to help diagnose Team Profile issues. [#1776](https://github.com/zowe/vscode-extension-for-zowe/pull/1776)
```
Comment (JillieBeanSim) [inline packages/zowe-explorer/CHANGELOG.md:12]: would the added/improved logging be `Added` vs `BugFix`?

Comment (JillieBeanSim) [reply packages/zowe-explorer/CHANGELOG.md:12]: if we do mark it `Added` could we move it to the top of the list too?

Comment (zFernand0) [reply packages/zowe-explorer/CHANGELOG.md]: moved to the top, thanks! 😋 
As for the keyword, I'm happy to revisit it here: https://github.com/zowe/vscode-extension-for-zowe/issues/1791#issuecomment-1122701379

**Review by JillieBeanSim — APPROVED** [review]
LGTM! thanks @zFernand0

**Comment by zFernand0** [comment]
Please see verified commit for `ze-build` results (6f96467a77a125d9161c81cc811ad2b146cfbe27)
image

**Review by rudyflores — APPROVED** [review]
LGTM!

**Pull request merged**